docs/runtime_CLI.md

```diff
@@ -0,0 +1,7 @@
+### BMv2 meter rate and burst_size units
```

Comment:
I think if we are going to start a CLI documentation we should do it right and organize it by command. For the `meter_set_rates` command, more information may be useful, and in particular a pointer to RFC 2698.